x-pack/solutions/security/plugins/security_solution/public/siem_migrations/rules/service/rule_migrations_service.ts

```diff
@@ -144,6 +148,21 @@ export class SiemRulesMigrationsService {
     }
   }
 
+  public async deleteMigration(migrationId: string): Promise<string> {
+    try {
```

Comment:
@semd , Don't we need capabilities check here?

Reply:
We don't have capabilities checks in any of the operations in the service, nor in the API functions that are sometimes called directly.

We assume the UI components are checking them and showing the proper messages. The only place we do the check in the service is the automatic migration start (for `interrupted` migrations), but it makes sense because it is triggered automatically without any explicit action by the user in the UI.
If we consider it should be checked here, we'd need to add the check to every operation, which I don't think is necessary. The privilege checks in the UI are just informative.
The API is where the actions should be strictly restricted by privileges (and sometimes they are not...).

I think we are fine here.

Reply:
Okay got it.. I saw it `startRuleMigration` and thought it was needed here as well but i understand your explanation.